game-cards/SGgame.js

```diff
@@ -373,22 +373,30 @@ exports.commands = {
 	confirmpickstarter: 'pickstarter',
 	pickstarter: function (target, room, user, connection, cmd) {
 		if (!user.console || user.console.gameId !== 'SGgame') return;
-		let starters = ['Bulbasaur', 'Chikorita', 'Treecko', 'Turtwig', 'Snivy', 'Chespin', 'Rowlet', 'Charmander', 'Cyndaquil', 'Torchic', 'Chimchar', 'Tepig', 'Fennekin', 'Litten', 'Squirtle', 'Totodile', 'Mudkip', 'Piplup', 'Oshawott', 'Froakie', 'Popplio'];
+		let starters = ['Bulbasaur', 'Chikorita', 'Treecko', 'Turtwig', 'Snivy', 'Chespin', 'Rowlet', 'Charmander', 'Cyndaquil', 'Torchic', 'Chimchar', 'Tepig', 'Fennekin', 'Litten', 'Squirtle', 'Totodile', 'Mudkip', 'Piplup', 'Oshawott', 'Froakie', 'Popplio', 'Pikachu', 'Eevee'];
 		if (!target || starters.indexOf(target) === -1) return false;
 		let type, typeColor;
 		if (starters.indexOf(target) <= 6) {
 			type = "Grass";
 		} else if (starters.indexOf(target) >= 7 && starters.indexOf(target) <= 13) {
 			type = "Fire";
-		} else if (starters.indexOf(target) >= 14) {
+		} else if (starters.indexOf(target) >= 14 && starters.indexOf(target) <= 20) {
 			type = "Water";
+		} else if (starters.indexOf(target) === 21) {
+			type = "Electric";
+		} else {
+			type = "Normal";
 		}
 		if (type === "Grass") {
```

Comment:
You see the checks that start on line 390? `if (type === "Grass")`?

remove then and merge thier typeColor parts into the checks that get the type